pulsar_spectra/catalogue_papers/Kramer_1999.yaml

```diff
@@ -1,20 +1,22 @@
 {
+ "J0437-4715": {
+  "Frequency MHz": [],
+  "Bandwidth MHz": [],
+  "Flux Density mJy": [],
+  "Flux Density error mJy": []
+ },
  "J0621+1002": {
   "Frequency MHz": [
-   2695.0,
-   4850.0
+   2695.0
   ],
   "Bandwidth MHz": [
-   80.0,
    80.0
   ],
   "Flux Density mJy": [
-   0.3,
-   -1.9
+   0.3
   ],
   "Flux Density error mJy": [
-   0.2,
-   0.3
+   0.2
   ]
  },
  "J0751+1807": {
```

Comment:
I think you were supposed to put these new values into J0437

Reply:
Yes, seems I missed this one.

Reply:
"J0437-4715": {
  "Frequency MHz": [
   4850.0
  ],
  "Bandwidth MHz": [
   80.0
  ],
  "Flux Density mJy": [
   8.5
  ],
  "Flux Density error mJy": [
   2.5
  ]
 }